package.json

```diff
@@ -77,8 +78,8 @@
     "url": "git+https://github.com/mozilla/blurts-server.git"
   },
   "scripts": {
-    "start": "npm run build & node server.js",
-    "dev": "npm run build & nodemon server.js",
+    "start": "source .env && if [[ $MONITOR_V2 == 'true' ]]; then npm start --workspace=src; else npm run build & node server.js; fi;",
```

Comment:
For these two commands, an option is to have two separate .env files (one for the old system, living in root, and another for the new version, in /src)

In that case, we will `source .env` separately based on $MONITOR_V2

Reply:
A bit of chicken and the egg there – this code is sourcing .env in order to get `MONITOR_V2` in the first place 😅

Reply:
That's true, nvm